images/landing-page/src/hooks/use-urls.js

```diff
@@ -26,7 +26,7 @@ const toTitleCase = (phrase) => {
     .join(" ");
 };
 
-const fetchParameters = async (groups, jwt) => {
+const fetchParameters = async (groups, idToken) => {
```

Comment:
Changed the naming here to reflect what we actually pass into the token validation lambda.  Was confusing and wasn't sure what to pass in at first.

Can change back if there's a good reason for just `jwt` naming